src/server.cpp

```diff
@@ -351,6 +351,7 @@ Server::~Server()
 	// Deinitialize scripting
 	infostream << "Server: Deinitializing scripting" << std::endl;
 	delete m_script;
+	delete Settings::getLayer(SL_GAME);
```

Comment:
layers are not automatically deleted in `~Settings()`?

Reply:
Yes, but this object is created in `loadGameConfAndInitWorld` (subgames.cpp) which could be changed so that the ownership is transferred to `Server`.
I did not change that yet to keep the diff small as possible.